recipes-docs/rzg2l-readme/rzg2l-sbc-readme_1.0.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "Support build script for RZG2L-SBC"
```

Comment:
The summary is not correct, please adjust it.

Reply:
Updated in the commit: fa4a70cd5eb6037a6490416b3944b4b8ce0fae77